include/oneapi/tbb/detail/_flow_graph_cache_impl.h

```diff
@@ -125,6 +136,16 @@ class predecessor_cache : public node_cache< sender<T>, M > {
         } while ( msg == false );
         return msg;
     }
+public:
+    bool get_item( output_type& v ) {
+        return get_item_impl(v);
+    }
+
+#if __TBB_PREVIEW_FLOW_GRAPH_TRY_PUT_AND_WAIT
+    bool get_item( output_type& v, message_metainfo& metainfo ) {
+        return get_item_impl(v, &metainfo);
+    }
+#endif
```

Comment:
Does it make sense to make it a single method as `get_item_impl`? (may be for consistency)

Reply:
I am not sure I understood correctly. `get_item` should be considered a public API for a cache. `get_item_impl` - as a private method. We need to keep the existing public API `get_item(v)` and add a new one `get_item(v, metainfo)` and implement both of them on top of `get_item_impl`.